feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+	"github.com/openconfig/gribigo/fluent"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/gnmi/oc/ocpath"
+	"github.com/openconfig/ondatra/netutil"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	ipv4PLen          = 30
+	ipv6PLen          = 126
+	isisInstance      = "DEFAULT"
+	dutAreaAddress    = "49.0001"
+	ateAreaAddress    = "49"
+	dutSysID          = "1920.0000.2001"
+	asn               = 64501
+	acceptRoutePolicy = "PERMIT-ALL"
+	trafficPPS        = 2500000
+	srcTrafficV4      = "100.0.1.1"
+	srcTrafficV6      = "2002:db8:64:64::1"
+	dstTrafficV4      = "100.0.2.1"
+	dstTrafficV6      = "2003:db8:64:64::1"
+	v4Count           = 254
+	v6Count           = 100000000
+	lagTypeLACP       = oc.IfAggregate_AggregationType_LACP
+	ieee8023adLag     = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	ethernetCsmacd    = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	LAG1              = "lag1"
+	LAG2              = "lag2"
+	LAG3              = "lag3"
+)
+
+type aggPortData struct {
+	dutIPv4      string
+	ateIPv4      string
+	dutIPv6      string
+	ateIPv6      string
+	ateAggName   string
+	ateAggMAC    string
+	ateISISSysID string
+	ateLagCount  uint32
+}
+
+type ipAddr struct {
+	ip     string
+	prefix uint32
+}
+
+// testArgs holds the objects needed by a test case.
+type testArgs struct {
+	dut    *ondatra.DUTDevice
+	ate    *ondatra.ATEDevice
+	top    gosnappi.Config
+	ctx    context.Context
+	client *fluent.GRIBIClient
+}
+
+var (
+	agg1 = &aggPortData{
+		dutIPv4:      "192.0.2.1",
+		ateIPv4:      "192.0.2.2",
+		dutIPv6:      "2001:db8::1",
+		ateIPv6:      "2001:db8::2",
+		ateAggName:   LAG1,
+		ateAggMAC:    "02:00:01:01:01:01",
+		ateISISSysID: "640000000002",
+		ateLagCount:  1,
+	}
+	agg2 = &aggPortData{
+		dutIPv4:      "192.0.2.5",
+		ateIPv4:      "192.0.2.6",
+		dutIPv6:      "2001:db8::5",
+		ateIPv6:      "2001:db8::6",
+		ateAggName:   LAG2,
+		ateAggMAC:    "02:00:01:01:02:01",
+		ateISISSysID: "640000000003",
+		ateLagCount:  2,
+	}
+	agg3 = &aggPortData{
+		dutIPv4:      "192.0.2.9",
+		ateIPv4:      "192.0.2.10",
+		dutIPv6:      "2001:db8::9",
+		ateIPv6:      "2001:db8::a",
+		ateAggName:   LAG3,
+		ateAggMAC:    "02:00:01:01:03:01",
+		ateISISSysID: "640000000004",
+		ateLagCount:  1,
+	}
+
+	dutLoopback = attrs.Attributes{
+		Desc:    "Loopback ip",
+		IPv4:    "192.0.2.21",
+		IPv6:    "2001:db8::21",
+		IPv4Len: 32,
+		IPv6Len: 128,
+	}
+
+	pfx1AdvV4                = &ipAddr{ip: "100.0.1.1", prefix: 24}
+	pfx1AdvV6                = &ipAddr{ip: "2002:db8:64:64::1", prefix: 64}
+	pfx2AdvV4                = &ipAddr{ip: "100.0.2.1", prefix: 24}
+	pfx2AdvV6                = &ipAddr{ip: "2003:db8:64:64::1", prefix: 64}
+	pfx3AdvV4                = &ipAddr{ip: "100.0.3.1", prefix: 24}
+	pfx4AdvV4                = &ipAddr{ip: "100.0.4.1", prefix: 24}
```

Comment:
100.0.4.1/24 is not valid prefix so gribi route add wil lfail, please change to 100.0.4.1 to 100.0.4.0 . similarly check other prefixes as well i.e pfx1AdvV4,pfx2AdvV4,pfx3AdvV4,

Reply:
I have tested and the same is working. FYI below:
 I L2     100.0.1.0/24 [115/30] via 192.0.2.2, Port-Channel3
 I L2     100.0.2.0/24 [115/30] via 192.0.2.6, Port-Channel4, weight link bandwidth
                                via 192.0.2.10, Port-Channel5, weight link bandwidth
 B I      100.0.3.0/24 [200/0] via 192.0.2.6, Port-Channel4, weight link bandwidth
                               via 192.0.2.10, Port-Channel5, weight link bandwidth
 G        100.0.4.0/24 [5/0] via 192.0.2.6, Port-Channel4

Anyway i am changing it to as you suggested